### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by MasterGamer64** [PrOF-kk/rainmeter-gkeep#5]
When i try to type the app password with the email in the box it says this.
If i try to do it via the settings it just gives me the windows to login and does nothing.
![python_d50bswL0Pc](https://user-images.githubusercontent.com/83783476/192154858-5c8116e7-61c5-4d77-884d-bc1da1b32d14.png)

**Comment by PrOF-kk** [comment]
Do you have 2FA enabled? If so, are you using an [App Password](https://support.google.com/accounts/answer/185833) (and not your regular password)?

**Comment by MasterGamer64** [comment]
Yes i have 2fa and i am using a app password. Since then i found that its a issue with the gkeepapi is there any way to fix it ?

**Comment by PrOF-kk** [comment]
Sorry, I don't know how to help. Yes it's likely https://github.com/kiwiz/gkeepapi/issues/81

**Comment by BIRDIE13** [comment]
> Sorry, I don't know how to help. Yes it's likely [kiwiz/gkeepapi#81](https://github.com/kiwiz/gkeepapi/issues/81)

Has this been fixed or is the project dead? I get prompted with this but nothing happens when i try to login
[image](https://github.com/PrOF-kk/rainmeter-gkeep/assets/37814738/99d1debc-f3c4-445e-8f2c-54b4dbb842b0)

**Comment by PrOF-kk** [comment]
If you don't get an error message at all it might be a different issue. Otherwise, I can't do anything about it since it's an issue with the library this project uses.

**Comment by maaaaaajin** [comment]
Is the problem solved by now? Got the same bad authentication error.

**Comment by PrOF-kk** [comment]
It's not, otherwise this issue and https://github.com/kiwiz/gkeepapi/issues/81 would be closed.

**Comment by MasterGamer64** [comment]
It seems that the library was updated but requires changing the login method on the app side. This might be worth looking into.
